### PR TITLE
Adding LLM tuning guide

### PR DESCRIPTION
**Pull request by maaquib**
## Description ##

Adding a new guide for tunable parameters for all the supported LLM backends for **0.23.0** release

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Comment by lanking520** [comment]
could we also make a copy in master?

**Review by lanking520 — APPROVED** [review]

File: serving/docs/configurations_large_model_inference_containers.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+| Item                                    | Required | Description                                                                                                                                                                        | Example value                                            |
+|-----------------------------------------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------|
+| option.task                             | No       | The task used in Hugging Face for different pipelines.                                                                                                                             | `text-generation`                                        |
+| option.device_id                        | No       | Load model on a specific device. Do not set this if you use option.tensor_parallel_device.                                                                                         | `0, 1`                                                   |
```
Comment (frankfliu) [inline serving/docs/configurations_large_model_inference_containers.md:38]: We don't have this option, are you referring to `load_on_devices`?
device_id is used internally and should not be set manually.

File: serving/docs/configurations_large_model_inference_containers.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+|---------------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------|
+| option.task               | No       | The task used in Hugging Face for different pipelines.                                                                                                                                          | `text-generation`              |
+| option.max_tokens         | No       | Total number of tokens (input and output) with which DeepSpeed can work. The number of output tokens in the difference between the total number of tokens and the number of input tokens.       | `1024`                         |
+| option.low_cpu_mem_usage  | No       | Reduce CPU memory usage when loading models. We recommend that you set this to True.                                                                                                            | `True`                         |
```
Comment (frankfliu) [inline serving/docs/configurations_large_model_inference_containers.md:22]: Let use lower case for all True/False value as example in the serving.properties

File: serving/docs/configurations_large_model_inference_containers.md (hunk 1 above)
Comment (maaquib) [reply serving/docs/configurations_large_model_inference_containers.md]: Replaced. This is documented in `Hugging Face handler settings` at the [configuration](https://docs.aws.amazon.com/sagemaker/latest/dg/large-model-inference-configuration.html) docs

File: serving/docs/configurations_large_model_inference_containers.md (hunk 2 above)
Comment (maaquib) [reply serving/docs/configurations_large_model_inference_containers.md]: Done

**Comment by maaquib** [comment]
> could we also make a copy in master?

Will push to master branch once this PR is approved and merged

**Review by frankfliu — APPROVED** [review]

**Pull request merged**